frontend/src/app/modules/work-package-graphs/configuration-modal/tabs/filters-tab-inner.component.ts

```diff
@@ -6,7 +6,6 @@ import {WorkPackageViewFiltersService} from 'core-app/modules/work_packages/rout
 import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {WpGraphConfigurationService} from "core-app/modules/work-package-graphs/configuration/wp-graph-configuration.service";
 import {WorkPackageStatesInitializationService} from "core-components/wp-list/wp-states-initialization.service";
-import {QueryResource} from "core-app/modules/hal/resources/query-resource";
 import {QuerySpacedTabComponent} from "core-app/modules/work-package-graphs/configuration-modal/tabs/abstract-query-spaced-tab.component";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c9bc013cc27eb3bb23aea9c85aaedb5b868ef858/frontend/src/app/modules/work-package-graphs/configuration-modal/tabs/filters-tab-inner.component.ts#L10"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>